bitnami/tomcat/templates/pvc.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.persistence.enabled -}}
+{{- if and .Values.persistence.enabled (not .Values.persistence.existingClaim) -}}
 kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
```

Comment:
I can't comment in the next line because GitHub considers it out of the scope of this PR, but now the name of the PVC should be something like:
```
  name: {{ template "tomcat.pvc" . }}
```

Reply:
No it shouldn't, `tomcat.pvc` is for deciding whether it should use the one created by the chart or an already existing one.